slapp/explorer/results.py

```diff
@@ -68,6 +71,28 @@ def get_sensitivity_result(sensitivity: str, region: str, technology: str) -> di
     return results
 
 
+def get_alternative_result_for_region(region: str) -> dict:
+    """Return Alternative Results for ranges by region."""
+    results = AlternativeResult.objects.filter(region=region).select_related("alternative")
```

Comment:
I would have divergence as addtional parameter in function. 
Then this query could be done simpler by filtering Alternative objects for given divergence and filtering result set by region which should make filtering in for loop unnecessary